sdk/loadtestservice/azure-developer-loadtesting/README.md

```diff
@@ -16,20 +16,24 @@ python -m pip install azure-developer-loadtesting
 - Python 3.7 or later is required to use this package.
 - You need an [Azure subscription][azure_sub] to use this package.
 - An existing Azure Developer LoadTesting instance.
+
 #### Create with an Azure Active Directory Credential
+
 To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
 provide an instance of the desired credential type obtained from the
 [azure-identity][azure_identity_credentials] library.
 
 To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip]
 
 After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
-As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
 
-Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
-`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`
+As an example, sign in via the Azure CLI `az login` command and DefaultAzureCredential will authenticate as that user.
```

Comment:
should still link to what a `DefaultAzureCredential` is

Reply:
updated in: https://github.com/Azure/azure-sdk-for-python/pull/25550/commits/91cda2cc1bc33e23cfce78efafdac3413c399ada